VR-5194 model find

evgeniilezhnin opened the pull request.

evgeniilezhnin committed:
VR-5194 model find

convoliution commented on client/verta/verta/_registry/models.py:
```diff
@@ -33,3 +45,17 @@ def with_workspace(self, workspace_name=None):
+        return msg.pagination.page_limit
+
+    def page_number(self, msg):
+        return msg.pagination.page_number
```
Interesting, I didn't realize these APIs used a nested pagination. I will need to follow up on this later to use a cleaner interface.

convoliution reviewed (approved):
Implementation looks good. Please add some tests, especially once the labels PR is also added. We'd like to cover cases such as
```python
client.registered_models.find(["labels == \"foo\"])
```

evgeniilezhnin committed:
Merge branch 'el/VR-5193_registry_model_labels' into el/VR-5194_model_find

evgeniilezhnin committed:
VR-5194 rename tags to labels, find test

evgeniilezhnin commented:
> Implementation looks good. Please add some tests, especially once the labels PR is also added. We'd like to cover cases such as
> 
> ```python
> client.registered_models.find(["labels == \"foo\"])
> ```

added

evgeniilezhnin committed:
Merge branch 'el/VR-5193_registry_model_labels' into el/VR-5194_model_find

evgeniilezhnin committed:
Merge branch 'el/VR-5193_registry_model_labels' into el/VR-5194_model_find

evgeniilezhnin committed:
Merge branch 'el/VR-5193_registry_model_labels' into el/VR-5194_model_find

evgeniilezhnin committed:
Merge branch 'el/VR-5194_model_find' of github.com:VertaAI/modeldb into el/VR-5194_model_find

evgeniilezhnin committed:
VR-5194 labels param
